app/uk/gov/hmrc/cataloguefrontend/metrics/model/MetricsResponse.scala

```diff
@@ -16,10 +16,10 @@
 
 package uk.gov.hmrc.cataloguefrontend.metrics.model
 
-import play.api.libs.json.{Json, Reads}
+import play.api.libs.json.Reads
+
+final case class MetricsResponse(metrics: Seq[ServiceProgressMetrics])
 
-case class MetricsResponse(metrics: Seq[ServiceProgressMetrics])
 object MetricsResponse {
-  implicit val reads: Reads[MetricsResponse] = Json.reads[MetricsResponse]
+  implicit val reads: Reads[MetricsResponse] = implicitly[Reads[Seq[ServiceProgressMetrics]]].map(MetricsResponse.apply)
```

Comment:
Should this be implicit on the companion object, since it only applies to the `MetricsConnector`? Also couples the model field name "metrics" to the external API.

Reply:
> Should this be implicit on the companion object, since it only applies to the MetricsConnector?

we had this discussion. while I appreciate your point about coherent vs incoherent type classes, we have already limited this type to the `metrics` package. so not sure if we should worry about models being reused. 

- as for the naming `MetricsResponse`, in the metrics service, I split the models into: 
- internal API
- external API
- domain model

didn't bother with all that here. but to clarify it's actually the response we get from the metrics service, as opposed to the response we expose to the catalogue-frontend UI. it's just a wrapper type for the array. (I seem to have forgotten to add it as an `AnyVal`)  what do you suggest we call it?